espnet2/enh/extractor/td_speakerbeam_extractor.py

```diff
@@ -118,10 +140,18 @@ def forward(
         B, L, N = feature.shape
 
         feature = feature.transpose(1, 2)  # B, N, L
-        aux_feature = aux_feature.transpose(1, 2)  # B, N, L'
+        if self.use_spk_emb:
+            # B, N, L'=1
+            if input_aux.dim() == 2:
```

Comment:
Is this expected to use `input_aux` instead of `aux_feature` here and after?

Reply:
Oh I think it should be `aux_feature`, although usually `input_aux` is equivalent here.